packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/dataservice/SimulationProxy.java

```diff
@@ -15,6 +16,30 @@
 @Produces(MediaType.APPLICATION_JSON)
 public interface SimulationProxy {
 
+	@GET
+	@Path("/{id}")
+	Response getSimulation(
```

Comment:
Rather than `Response`, should these return a `Simulation`? Looking at the front end it looks like that is what is being expected.  For just generic responses that we don't really care about we can probably avoid using POJOs but when we have them made may as well use them :)

Reply:
(This goes for any of the other endpoints here - I'm not sure what is being returned from these end points or what java objects we have but if we have them lets use them)